js/web/lib/onnxjs/execution-plan.ts

```diff
@@ -114,8 +106,7 @@ export class ExecutionPlan {
           return result;
         };
 
-        const outputList = isWebGLBackend ? await this.profiler.event('node', thisOp.node.name, execNodeFn, glCtx) :
-                                            await this.profiler.event('node', thisOp.node.name, execNodeFn);
+        const outputList = await this.profiler.event('node', thisOp.node.name, execNodeFn);
```

Comment:
no longer need execNodeFn now

Reply:
it makes the code slightly more readable, so let's keep it.